config/x86/kernel-config

```diff
@@ -504,7 +502,7 @@ CONFIG_HIBERNATION_SNAPSHOT_DEV=y
 CONFIG_PM_STD_PARTITION=""
 CONFIG_PM_SLEEP=y
 CONFIG_PM_SLEEP_SMP=y
-# CONFIG_PM_AUTOSLEEP is not set
+CONFIG_PM_AUTOSLEEP=y
```

Comment:
I'm quite sure that we don't need support sleep for BurmillaOS so probably it makes sense to leave this untouched 
and also set CONFIG_PM_SLEEP and CONFIG_PM_SLEEP_SMP to "n"
https://github.com/torvalds/linux/blob/master/kernel/power/Kconfig#L117-L145